packages/core/src/fhir-to-cda/cda-templates/components/__tests__/medications.test.ts

```diff
@@ -0,0 +1,190 @@
+import { Bundle, Medication, MedicationStatement } from "@medplum/fhirtypes";
+import { uuidv7 } from "@metriport/core/util/uuid-v7";
+import { removeEmptyFields } from "../../clinical-document/clinical-document";
+import { xmlBuilder } from "../../clinical-document/shared";
+import { NOT_SPECIFIED } from "../../constants";
+import { buildMedications } from "../medications";
+import { createEmptyBundle, getPastDateInDifferentFormats } from "./shared";
+
+let medStmntId: string;
+let medId: string;
+let pastDateFhir: string;
+let pastDateHumanReadable: string | undefined;
+let pastDateXml: string | undefined;
+let bundle: Bundle;
+let expectedXml: string;
+let medicationStatement: MedicationStatement;
+let medication: Medication;
+
+beforeEach(() => {
+  medStmntId = uuidv7();
+  medId = uuidv7();
+  const { dateFhir, dateHumanReadable, dateXml } = getPastDateInDifferentFormats();
+  pastDateFhir = dateFhir;
+  pastDateHumanReadable = dateHumanReadable;
+  pastDateXml = dateXml;
+
+  medicationStatement = {
```

Comment:
I suggest we keep payloads out of tests, the `makeMedicalStatement()` pattern we've been following in other tests work well for this.

Reply:
💯 
Noticed the test was getting bulky and hard to work with! Thx for the reminder